xla/stream_executor/dnn.h

```diff
@@ -49,6 +49,10 @@ namespace Eigen {
 struct half;
 }  // namespace Eigen
 
+namespace cudnn_frontend::graph {
```

Comment:
It should be a pure virtual base `se::dnn::DnnGraph` and `CudnnGraph : public DnnGraph` in CUDA backend.

Reply:
Done.